src/Dialogs/Device/DeviceEditWidget.cpp

```diff
@@ -214,6 +224,8 @@ DeviceEditWidget::UpdateVisibilities() noexcept
   SetRowVisible(SyncToDevice, DeviceConfig::UsesDriver(type) &&
                 CanSendSettings(GetDataField(Driver)));
   SetRowAvailable(K6Bt, maybe_bluetooth);
+  SetRowVisible(EngineSensors, engine_sensor);
+  SetRowAvailable(EngineSensors, engine_sensor);
```

Comment:
Don't set both. Decide which one is the best choice here - I suggest using only SetRowAvailable